Docker fails to start: panic: runtime error: index out of range

<!--
If you are reporting a new issue, make sure that we do not have any duplicates
already open. You can ensure this by searching the issue list for this
repository. If there is a duplicate, please close your issue and add a comment
to the existing issue instead.

If you suspect your issue is a bug, please edit your issue description to
include the BUG REPORT INFORMATION shown below. If you fail to provide this
information within 7 days, we cannot debug your issue and will close it. We
will, however, reopen it if you later provide the information.

For more information about reporting issues, see
https://github.com/docker/docker/blob/master/CONTRIBUTING.md#reporting-other-issues

---------------------------------------------------
BUG REPORT INFORMATION
---------------------------------------------------
Use the commands below to provide key information from your environment:
You do NOT have to include this information if this is a FEATURE REQUEST
-->

**Output of `docker version`:**

```
Docker version 1.10.3, build 20f81dd
```


**Output of `docker daemon`:**

```
INFO[0001] Graph migration to content-addressability took 0.00 seconds
INFO[0001] Firewalld running: false
panic: runtime error: index out of range

goroutine 1 [running]:
panic(0xd8ff38, 0x1171e008)
        /home/theodor/resin-devices/resin-ts/build/tmp/sysroots/x86_64-linux/usr/lib/arm-poky-linux-gnueabi/go/src/runtime/panic.go:464 +0x330
github.com/boltdb/bolt.(*Bucket).pageNode(0x11711720, 0x6b720004, 0x0, 0x0, 0x0)
        /home/theodor/resin-devices/resin-ts/build/tmp/work/armv5e-poky-linux-gnueabi/docker/1.10.3+git20f81dde9bd97c86b2d0e33bbbf1388018611929-r0/git/vendor/src/github.com/boltdb/bolt/bucket.go:696 +0x2f8
github.com/boltdb/bolt.(*Cursor).search(0x11bf8eb4, 0x11bf8e1c, 0x1b, 0x1b, 0x6b720004, 0x0)
        /home/theodor/resin-devices/resin-ts/build/tmp/work/armv5e-poky-linux-gnueabi/docker/1.10.3+git20f81dde9bd97c86b2d0e33bbbf1388018611929-r0/git/vendor/src/github.com/boltdb/bolt/cursor.go:238 +0x30
github.com/boltdb/bolt.(*Cursor).seek(0x11bf8eb4, 0x11bf8e1c, 0x1b, 0x1b, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0, ...)
        /home/theodor/resin-devices/resin-ts/build/tmp/work/armv5e-poky-linux-gnueabi/docker/1.10.3+git20f81dde9bd97c86b2d0e33bbbf1388018611929-r0/git/vendor/src/github.com/boltdb/bolt/cursor.go:152 +0xd0
github.com/boltdb/bolt.(*Cursor).Seek(0x11bf8eb4, 0x11bf8e1c, 0x1b, 0x1b, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0)
        /home/theodor/resin-devices/resin-ts/build/tmp/work/armv5e-poky-linux-gnueabi/docker/1.10.3+git20f81dde9bd97c86b2d0e33bbbf1388018611929-r0/git/vendor/src/github.com/boltdb/bolt/cursor.go:111 +0x5c
github.com/docker/libkv/store/boltdb.(*BoltDB).List.func1(0x118f9f00, 0x0, 0x0)
        /home/theodor/resin-devices/resin-ts/build/tmp/work/armv5e-poky-linux-gnueabi/docker/1.10.3+git20f81dde9bd97c86b2d0e33bbbf1388018611929-r0/git/vendor/src/github.com/docker/libkv/store/boltdb/boltdb.go:285 +0x178
github.com/boltdb/bolt.(*DB).View(0x1178ec00, 0x11bf8f24, 0x0, 0x0)
        /home/theodor/resin-devices/resin-ts/build/tmp/work/armv5e-poky-linux-gnueabi/docker/1.10.3+git20f81dde9bd97c86b2d0e33bbbf1388018611929-r0/git/vendor/src/github.com/boltdb/bolt/db.go:578 +0xb0
github.com/docker/libkv/store/boltdb.(*BoltDB).List(0x1193f9c0, 0x11711620, 0x1b, 0x0, 0x0, 0x0, 0x0, 0x0)
        /home/theodor/resin-devices/resin-ts/build/tmp/work/armv5e-poky-linux-gnueabi/docker/1.10.3+git20f81dde9bd97c86b2d0e33bbbf1388018611929-r0/git/vendor/src/github.com/docker/libkv/store/boltdb/boltdb.go:299 +0x19c
github.com/docker/libnetwork/datastore.(*cache).kmap(0x11794670, 0xb54c6158, 0x1188bb60, 0x74ff8, 0x0, 0x0)
        /home/theodor/resin-devices/resin-ts/build/tmp/work/armv5e-poky-linux-gnueabi/docker/1.10.3+git20f81dde9bd97c86b2d0e33bbbf1388018611929-r0/git/vendor/src/github.com/docker/libnetwork/datastore/cache.go:43 +0x20c
github.com/docker/libnetwork/datastore.(*cache).list(0x11794670, 0xb54c6158, 0x1188bb60, 0x0, 0x0, 0x0, 0x0, 0x0)
        /home/theodor/resin-devices/resin-ts/build/tmp/work/armv5e-poky-linux-gnueabi/docker/1.10.3+git20f81dde9bd97c86b2d0e33bbbf1388018611929-r0/git/vendor/src/github.com/docker/libnetwork/datastore/cache.go:139 +0x48
github.com/docker/libnetwork/datastore.(*datastore).List(0x1170f170, 0x117115c0, 0x1b, 0xb54c6158, 0x1188bb60, 0x0, 0x0, 0x0, 0x0, 0x0)
        /home/theodor/resin-devices/resin-ts/build/tmp/work/armv5e-poky-linux-gnueabi/docker/1.10.3+git20f81dde9bd97c86b2d0e33bbbf1388018611929-r0/git/vendor/src/github.com/docker/libnetwork/datastore/datastore.go:472 +0xd8
github.com/docker/libnetwork/drivers/bridge.(*driver).populateNetworks(0x1170e1b0, 0x0, 0x0)
        /home/theodor/resin-devices/resin-ts/build/tmp/work/armv5e-poky-linux-gnueabi/docker/1.10.3+git20f81dde9bd97c86b2d0e33bbbf1388018611929-r0/git/vendor/src/github.com/docker/libnetwork/drivers/bridge/bridge_store.go:49 +0x130
github.com/docker/libnetwork/drivers/bridge.(*driver).initStore(0x1170e1b0, 0x1187de20, 0x0, 0x0)
        /home/theodor/resin-devices/resin-ts/build/tmp/work/armv5e-poky-linux-gnueabi/docker/1.10.3+git20f81dde9bd97c86b2d0e33bbbf1388018611929-r0/git/vendor/src/github.com/docker/libnetwork/drivers/bridge/bridge_store.go:42 +0x33c
github.com/docker/libnetwork/drivers/bridge.(*driver).configure(0x1170e1b0, 0x1187de20, 0x0, 0x0)
        /home/theodor/resin-devices/resin-ts/build/tmp/work/armv5e-poky-linux-gnueabi/docker/1.10.3+git20f81dde9bd97c86b2d0e33bbbf1388018611929-r0/git/vendor/src/github.com/docker/libnetwork/drivers/bridge/bridge.go:395 +0x310
github.com/docker/libnetwork/drivers/bridge.Init(0xb54c5f68, 0x118d5860, 0x1187de20, 0x0, 0x0)
        /home/theodor/resin-devices/resin-ts/build/tmp/work/armv5e-poky-linux-gnueabi/docker/1.10.3+git20f81dde9bd97c86b2d0e33bbbf1388018611929-r0/git/vendor/src/github.com/docker/libnetwork/drivers/bridge/bridge.go:140 +0x58c
github.com/docker/libnetwork.initDrivers(0x118d5860, 0x0, 0x0)
        /home/theodor/resin-devices/resin-ts/build/tmp/work/armv5e-poky-linux-gnueabi/docker/1.10.3+git20f81dde9bd97c86b2d0e33bbbf1388018611929-r0/git/vendor/src/github.com/docker/libnetwork/drivers.go:20 +0x220
github.com/docker/libnetwork.New(0x1187dca0, 0x5, 0x8, 0x0, 0x0, 0x0, 0x0)
        /home/theodor/resin-devices/resin-ts/build/tmp/work/armv5e-poky-linux-gnueabi/docker/1.10.3+git20f81dde9bd97c86b2d0e33bbbf1388018611929-r0/git/vendor/src/github.com/docker/libnetwork/controller.go:189 +0x464
github.com/docker/docker/daemon.(*Daemon).initNetworkController(0x118b0000, 0x11be2300, 0x0, 0x0, 0x0, 0x0)
        /home/theodor/resin-devices/resin-ts/build/tmp/work/armv5e-poky-linux-gnueabi/docker/1.10.3+git20f81dde9bd97c86b2d0e33bbbf1388018611929-r0/git/.gopath/src/github.com/docker/docker/daemon/daemon_unix.go:532 +0x9c
github.com/docker/docker/daemon.NewDaemon(0x11be2300, 0x1171e638, 0x0, 0x0, 0x0)
        /home/theodor/resin-devices/resin-ts/build/tmp/work/armv5e-poky-linux-gnueabi/docker/1.10.3+git20f81dde9bd97c86b2d0e33bbbf1388018611929-r0/git/.gopath/src/github.com/docker/docker/daemon/daemon.go:761 +0x1dfc
main.(*DaemonCli).CmdDaemon(0x11bd5ec0, 0x1170a108, 0x0, 0x0, 0x0, 0x0)
        /home/theodor/resin-devices/resin-ts/build/tmp/work/armv5e-poky-linux-gnueabi/docker/1.10.3+git20f81dde9bd97c86b2d0e33bbbf1388018611929-r0/git/docker/daemon.go:252 +0x1430
reflect.callMethod(0x119523a0, 0x11bf9e04)
        /home/theodor/resin-devices/resin-ts/build/tmp/sysroots/x86_64-linux/usr/lib/arm-poky-linux-gnueabi/go/src/reflect/value.go:631 +0x1ac
reflect.methodValueCall(0x1170a108, 0x0, 0x0, 0x1, 0x119523a0, 0x0, 0x0, 0x119523a0, 0x1171e4e8, 0x1efcc, ...)
        /home/theodor/resin-devices/resin-ts/build/tmp/sysroots/x86_64-linux/usr/lib/arm-poky-linux-gnueabi/go/src/reflect/asm_arm.s:29 +0x38
github.com/docker/docker/cli.(*Cli).Run(0x119522c0, 0x1170a108, 0x1, 0x1, 0x0, 0x0)
        /home/theodor/resin-devices/resin-ts/build/tmp/work/armv5e-poky-linux-gnueabi/docker/1.10.3+git20f81dde9bd97c86b2d0e33bbbf1388018611929-r0/git/.gopath/src/github.com/docker/docker/cli/cli.go:89 +0x2e8
main.main()
        /home/theodor/resin-devices/resin-ts/build/tmp/work/armv5e-poky-linux-gnueabi/docker/1.10.3+git20f81dde9bd97c86b2d0e33bbbf1388018611929-r0/git/docker/docker.go:63 +0x40c
```

**Additional information**
I have the exact same setup (OS + docker 1.10) on a few machines with different architectures (e.g. x86, armv6, armv7), but on a machine with armv5  I get the error mentioned above. After a bit of code digging it look like docker does not find the correct index for my network interface (the id should be smaller id - 1).

Any hints will be heavily appreciated.

Here is the strace output:

`execve("./docker-1.11.0", ["./docker-1.11.0", "daemon"], [/* 15 vars */]) = 0
brk(0)                                  = 0x159e000
uname({sysname="Linux", nodename="ts-pxa", ...}) = 0
access("/etc/ld.so.preload", R_OK)      = -1 ENOENT (No such file or directory)
open("/usr/lib/tls/v5l/libpthread.so.0", O_RDONLY|O_CLOEXEC) = -1 ENOENT (No such file or directory)
stat64("/usr/lib/tls/v5l", 0xbea896d8)  = -1 ENOENT (No such file or directory)
open("/usr/lib/tls/libpthread.so.0", O_RDONLY|O_CLOEXEC) = -1 ENOENT (No such file or directory)
stat64("/usr/lib/tls", 0xbea896d8)      = -1 ENOENT (No such file or directory)
open("/usr/lib/v5l/libpthread.so.0", O_RDONLY|O_CLOEXEC) = -1 ENOENT (No such file or directory)
stat64("/usr/lib/v5l", 0xbea896d8)      = -1 ENOENT (No such file or directory)
open("/usr/lib/libpthread.so.0", O_RDONLY|O_CLOEXEC) = -1 ENOENT (No such file or directory)
stat64("/usr/lib", {st_mode=S_IFDIR|0755, st_size=4096, ...}) = 0
open("/lib/tls/v5l/libpthread.so.0", O_RDONLY|O_CLOEXEC) = -1 ENOENT (No such file or directory)
stat64("/lib/tls/v5l", 0xbea896d8)      = -1 ENOENT (No such file or directory)
open("/lib/tls/libpthread.so.0", O_RDONLY|O_CLOEXEC) = -1 ENOENT (No such file or directory)
stat64("/lib/tls", 0xbea896d8)          = -1 ENOENT (No such file or directory)
open("/lib/v5l/libpthread.so.0", O_RDONLY|O_CLOEXEC) = -1 ENOENT (No such file or directory)
stat64("/lib/v5l", 0xbea896d8)          = -1 ENOENT (No such file or directory)
open("/lib/libpthread.so.0", O_RDONLY|O_CLOEXEC) = 3
read(3, "\177ELF\1\1\1\3\0\0\0\0\0\0\0\0\3\0(\0\1\0\0\0\370J\376O4\0\0\0"..., 512) = 512
fstat64(3, {st_mode=S_IFREG|0755, st_size=99060, ...}) = 0
mmap2(NULL, 4096, PROT_READ|PROT_WRITE, MAP_PRIVATE|MAP_ANONYMOUS, -1, 0) = 0xb6f23000
mmap2(0x4ffe0000, 168836, PROT_READ|PROT_EXEC, MAP_PRIVATE|MAP_DENYWRITE, 3, 0) = 0x4ffe0000
mprotect(0x4fff7000, 61440, PROT_NONE)  = 0
mmap2(0x50006000, 8192, PROT_READ|PROT_WRITE, MAP_PRIVATE|MAP_FIXED|MAP_DENYWRITE, 3, 0x16000) = 0x50006000
mmap2(0x50008000, 4996, PROT_READ|PROT_WRITE, MAP_PRIVATE|MAP_FIXED|MAP_ANONYMOUS, -1, 0) = 0x50008000
close(3)                                = 0
open("/usr/lib/libdl.so.2", O_RDONLY|O_CLOEXEC) = -1 ENOENT (No such file or directory)
open("/lib/libdl.so.2", O_RDONLY|O_CLOEXEC) = 3
read(3, "\177ELF\1\1\1\0\0\0\0\0\0\0\0\0\3\0(\0\1\0\0\0(\t\10A4\0\0\0"..., 512) = 512
fstat64(3, {st_mode=S_IFREG|0755, st_size=11224, ...}) = 0
mmap2(0x41080000, 73912, PROT_READ|PROT_EXEC, MAP_PRIVATE|MAP_DENYWRITE, 3, 0) = 0x41080000
mprotect(0x41082000, 61440, PROT_NONE)  = 0
mmap2(0x41091000, 8192, PROT_READ|PROT_WRITE, MAP_PRIVATE|MAP_FIXED|MAP_DENYWRITE, 3, 0x1000) = 0x41091000
close(3)                                = 0
open("/usr/lib/libdevmapper.so.1.02", O_RDONLY|O_CLOEXEC) = 3
read(3, "\177ELF\1\1\1\0\0\0\0\0\0\0\0\0\3\0(\0\1\0\0\0\4c\0\0004\0\0\0"..., 512) = 512
fstat64(3, {st_mode=S_IFREG|0555, st_size=326364, ...}) = 0
mmap2(NULL, 394728, PROT_READ|PROT_EXEC, MAP_PRIVATE|MAP_DENYWRITE, 3, 0) = 0xb6ec2000
mprotect(0xb6f0f000, 61440, PROT_NONE)  = 0
mmap2(0xb6f1e000, 16384, PROT_READ|PROT_WRITE, MAP_PRIVATE|MAP_FIXED|MAP_DENYWRITE, 3, 0x4c000) = 0xb6f1e000
mmap2(0xb6f22000, 1512, PROT_READ|PROT_WRITE, MAP_PRIVATE|MAP_FIXED|MAP_ANONYMOUS, -1, 0) = 0xb6f22000
close(3)                                = 0
open("/usr/lib/libc.so.6", O_RDONLY|O_CLOEXEC) = -1 ENOENT (No such file or directory)
open("/lib/libc.so.6", O_RDONLY|O_CLOEXEC) = 3
read(3, "\177ELF\1\1\1\0\0\0\0\0\0\0\0\0\3\0(\0\1\0\0\0\214\201\352O4\0\0\0"..., 512) = 512
fstat64(3, {st_mode=S_IFREG|0755, st_size=1259332, ...}) = 0
mmap2(0x4fe90000, 1325408, PROT_READ|PROT_EXEC, MAP_PRIVATE|MAP_DENYWRITE, 3, 0) = 0x4fe90000
mprotect(0x4ffbf000, 61440, PROT_NONE)  = 0
mmap2(0x4ffce000, 16384, PROT_READ|PROT_WRITE, MAP_PRIVATE|MAP_FIXED|MAP_DENYWRITE, 3, 0x12e000) = 0x4ffce000
mmap2(0x4ffd2000, 6496, PROT_READ|PROT_WRITE, MAP_PRIVATE|MAP_FIXED|MAP_ANONYMOUS, -1, 0) = 0x4ffd2000
close(3)                                = 0
open("/usr/lib/libudev.so.1", O_RDONLY|O_CLOEXEC) = -1 ENOENT (No such file or directory)
open("/lib/libudev.so.1", O_RDONLY|O_CLOEXEC) = 3
read(3, "\177ELF\1\1\1\0\0\0\0\0\0\0\0\0\3\0(\0\1\0\0\0\0\0\4A4\0\0\0"..., 512) = 512
fstat64(3, {st_mode=S_IFREG|0755, st_size=119360, ...}) = 0
mmap2(0x41040000, 121600, PROT_READ|PROT_EXEC, MAP_PRIVATE|MAP_DENYWRITE, 3, 0) = 0x41040000
mmap2(0x4105c000, 8192, PROT_READ|PROT_WRITE, MAP_PRIVATE|MAP_FIXED|MAP_DENYWRITE, 3, 0x1b000) = 0x4105c000
close(3)                                = 0
open("/usr/lib/librt.so.1", O_RDONLY|O_CLOEXEC) = -1 ENOENT (No such file or directory)
open("/lib/librt.so.1", O_RDONLY|O_CLOEXEC) = 3
read(3, "\177ELF\1\1\1\3\0\0\0\0\0\0\0\0\3\0(\0\1\0\0\0p\27\6A4\0\0\0"..., 512) = 512
fstat64(3, {st_mode=S_IFREG|0755, st_size=28368, ...}) = 0
mmap2(NULL, 4096, PROT_READ|PROT_WRITE, MAP_PRIVATE|MAP_ANONYMOUS, -1, 0) = 0xb6ec1000
mmap2(0x41060000, 90720, PROT_READ|PROT_EXEC, MAP_PRIVATE|MAP_DENYWRITE, 3, 0) = 0x41060000
mprotect(0x41066000, 61440, PROT_NONE)  = 0
mmap2(0x41075000, 8192, PROT_READ|PROT_WRITE, MAP_PRIVATE|MAP_FIXED|MAP_DENYWRITE, 3, 0x5000) = 0x41075000
close(3)                                = 0
open("/usr/lib/libm.so.6", O_RDONLY|O_CLOEXEC) = -1 ENOENT (No such file or directory)
open("/lib/libm.so.6", O_RDONLY|O_CLOEXEC) = 3
read(3, "\177ELF\1\1\1\0\0\0\0\0\0\0\0\0\3\0(\0\1\0\0\0\260<\nA4\0\0\0"..., 512) = 512
fstat64(3, {st_mode=S_IFREG|0755, st_size=629804, ...}) = 0
mmap2(0x410a0000, 692452, PROT_READ|PROT_EXEC, MAP_PRIVATE|MAP_DENYWRITE, 3, 0) = 0x410a0000
mprotect(0x41139000, 61440, PROT_NONE)  = 0
mmap2(0x41148000, 8192, PROT_READ|PROT_WRITE, MAP_PRIVATE|MAP_FIXED|MAP_DENYWRITE, 3, 0x98000) = 0x41148000
close(3)                                = 0
open("/usr/lib/libresolv.so.2", O_RDONLY|O_CLOEXEC) = -1 ENOENT (No such file or directory)
open("/lib/libresolv.so.2", O_RDONLY|O_CLOEXEC) = 3
read(3, "\177ELF\1\1\1\0\0\0\0\0\0\0\0\0\3\0(\0\1\0\0\0\320&>A4\0\0\0"..., 512) = 512
fstat64(3, {st_mode=S_IFREG|0755, st_size=73716, ...}) = 0
mmap2(0x413e0000, 146048, PROT_READ|PROT_EXEC, MAP_PRIVATE|MAP_DENYWRITE, 3, 0) = 0x413e0000
mprotect(0x413f1000, 61440, PROT_NONE)  = 0
mmap2(0x41400000, 8192, PROT_READ|PROT_WRITE, MAP_PRIVATE|MAP_FIXED|MAP_DENYWRITE, 3, 0x10000) = 0x41400000
mmap2(0x41402000, 6784, PROT_READ|PROT_WRITE, MAP_PRIVATE|MAP_FIXED|MAP_ANONYMOUS, -1, 0) = 0x41402000
close(3)                                = 0
open("/usr/lib/libcap.so.2", O_RDONLY|O_CLOEXEC) = -1 ENOENT (No such file or directory)
open("/lib/libcap.so.2", O_RDONLY|O_CLOEXEC) = 3
read(3, "\177ELF\1\1\1\0\0\0\0\0\0\0\0\0\3\0(\0\1\0\0\0l\16\25A4\0\0\0"..., 512) = 512
fstat64(3, {st_mode=S_IFREG|0644, st_size=14476, ...}) = 0
mmap2(0x41150000, 77324, PROT_READ|PROT_EXEC, MAP_PRIVATE|MAP_DENYWRITE, 3, 0) = 0x41150000
mprotect(0x41153000, 61440, PROT_NONE)  = 0
mmap2(0x41162000, 4096, PROT_READ|PROT_WRITE, MAP_PRIVATE|MAP_FIXED|MAP_DENYWRITE, 3, 0x2000) = 0x41162000
close(3)                                = 0
open("/usr/lib/libgcc_s.so.1", O_RDONLY|O_CLOEXEC) = -1 ENOENT (No such file or directory)
open("/lib/libgcc_s.so.1", O_RDONLY|O_CLOEXEC) = 3
read(3, "\177ELF\1\1\1\0\0\0\0\0\0\0\0\0\3\0(\0\1\0\0\0\214\323\1A4\0\0\0"..., 512) = 512
fstat64(3, {st_mode=S_IFREG|0644, st_size=125436, ...}) = 0
mmap2(0x41010000, 188028, PROT_READ|PROT_EXEC, MAP_PRIVATE|MAP_DENYWRITE, 3, 0) = 0x41010000
mprotect(0x4102e000, 61440, PROT_NONE)  = 0
mmap2(0x4103d000, 4096, PROT_READ|PROT_WRITE, MAP_PRIVATE|MAP_FIXED|MAP_DENYWRITE, 3, 0x1d000) = 0x4103d000
close(3)                                = 0
mmap2(NULL, 4096, PROT_READ|PROT_WRITE, MAP_PRIVATE|MAP_ANONYMOUS, -1, 0) = 0xb6ec0000
mmap2(NULL, 4096, PROT_READ|PROT_WRITE, MAP_PRIVATE|MAP_ANONYMOUS, -1, 0) = 0xb6ebf000
set_tls(0xb6ebf4c0, 0xb6ebfbb8, 0x4fe80050, 0xb6ebf4c0, 0x4fe80050) = 0
mprotect(0x4ffce000, 8192, PROT_READ)   = 0
mprotect(0x41400000, 4096, PROT_READ)   = 0
mprotect(0x41148000, 4096, PROT_READ)   = 0
mprotect(0x50006000, 4096, PROT_READ)   = 0
mprotect(0x41075000, 4096, PROT_READ)   = 0
mprotect(0x41091000, 4096, PROT_READ)   = 0
mprotect(0x4105c000, 4096, PROT_READ)   = 0
mprotect(0xb6f1e000, 4096, PROT_READ)   = 0
mprotect(0x4fe7f000, 4096, PROT_READ)   = 0
set_tid_address(0xb6ebf068)             = 19369
set_robust_list(0xb6ebf070, 12)         = 0
rt_sigaction(SIGRTMIN, {0x4ffe43ac, [], SA_RESTORER|SA_SIGINFO, 0x4febe800}, NULL, 8) = 0
rt_sigaction(SIGRT_1, {0x4ffe4494, [], SA_RESTORER|SA_RESTART|SA_SIGINFO, 0x4febe800}, NULL, 8) = 0
rt_sigprocmask(SIG_UNBLOCK, [RTMIN RT_1], NULL, 8) = 0
ugetrlimit(RLIMIT_STACK, {rlim_cur=8192*1024, rlim_max=RLIM_INFINITY}) = 0
getpid()                                = 19369
sched_getaffinity(0, 8192, {1})         = 4
mmap2(0x1600000, 806363136, PROT_NONE, MAP_PRIVATE|MAP_ANONYMOUS, -1, 0) = 0x1600000
mmap2(NULL, 262144, PROT_READ|PROT_WRITE, MAP_PRIVATE|MAP_ANONYMOUS, -1, 0) = 0xb6e7f000
clock_gettime(CLOCK_MONOTONIC, {56447, 129366841}) = 0
mmap2(0x11700000, 1048576, PROT_READ|PROT_WRITE, MAP_PRIVATE|MAP_FIXED|MAP_ANONYMOUS, -1, 0) = 0x11700000
mmap2(0x116f0000, 65536, PROT_READ|PROT_WRITE, MAP_PRIVATE|MAP_FIXED|MAP_ANONYMOUS, -1, 0) = 0x116f0000
mmap2(0x1600000, 4096, PROT_READ|PROT_WRITE, MAP_PRIVATE|MAP_FIXED|MAP_ANONYMOUS, -1, 0) = 0x1600000
mmap2(NULL, 65536, PROT_READ|PROT_WRITE, MAP_PRIVATE|MAP_ANONYMOUS, -1, 0) = 0xb6e6f000
clock_gettime(CLOCK_MONOTONIC, {56447, 132734503}) = 0
rt_sigprocmask(SIG_SETMASK, NULL, [], 8) = 0
clock_gettime(CLOCK_MONOTONIC, {56447, 135338242}) = 0
clock_gettime(CLOCK_MONOTONIC, {56447, 135922799}) = 0
sigaltstack(NULL, {ss_sp=0, ss_flags=SS_DISABLE, ss_size=0}) = 0
sigaltstack({ss_sp=0x11702000, ss_flags=0, ss_size=32720}, NULL) = 0
gettid()                                = 19369
rt_sigprocmask(SIG_SETMASK, [], NULL, 8) = 0
clock_gettime(CLOCK_MONOTONIC, {56447, 139703343}) = 0
rt_sigaction(SIGHUP, NULL, {SIG_DFL, [], 0}, 8) = 0
rt_sigaction(SIGHUP, {0x8c7c4, ~[], SA_STACK|SA_RESTART|SA_SIGINFO}, NULL, 8) = 0
rt_sigaction(SIGINT, NULL, {SIG_DFL, [], 0}, 8) = 0
rt_sigaction(SIGINT, {0x8c7c4, ~[], SA_STACK|SA_RESTART|SA_SIGINFO}, NULL, 8) = 0
rt_sigaction(SIGQUIT, NULL, {SIG_DFL, [], 0}, 8) = 0
rt_sigaction(SIGQUIT, {0x8c7c4, ~[], SA_STACK|SA_RESTART|SA_SIGINFO}, NULL, 8) = 0
rt_sigaction(SIGILL, NULL, {SIG_DFL, [], 0}, 8) = 0
rt_sigaction(SIGILL, {0x8c7c4, ~[], SA_STACK|SA_RESTART|SA_SIGINFO}, NULL, 8) = 0
rt_sigaction(SIGTRAP, NULL, {SIG_DFL, [], 0}, 8) = 0
rt_sigaction(SIGTRAP, {0x8c7c4, ~[], SA_STACK|SA_RESTART|SA_SIGINFO}, NULL, 8) = 0
rt_sigaction(SIGABRT, NULL, {SIG_DFL, [], 0}, 8) = 0
rt_sigaction(SIGABRT, {0x8c7c4, ~[], SA_STACK|SA_RESTART|SA_SIGINFO}, NULL, 8) = 0
rt_sigaction(SIGBUS, NULL, {SIG_DFL, [], 0}, 8) = 0
rt_sigaction(SIGBUS, {0x8c7c4, ~[], SA_STACK|SA_RESTART|SA_SIGINFO}, NULL, 8) = 0
rt_sigaction(SIGFPE, NULL, {SIG_DFL, [], 0}, 8) = 0
rt_sigaction(SIGFPE, {0x8c7c4, ~[], SA_STACK|SA_RESTART|SA_SIGINFO}, NULL, 8) = 0
rt_sigaction(SIGUSR1, NULL, {SIG_DFL, [], 0}, 8) = 0
rt_sigaction(SIGUSR1, {0x8c7c4, ~[], SA_STACK|SA_RESTART|SA_SIGINFO}, NULL, 8) = 0
rt_sigaction(SIGSEGV, NULL, {SIG_DFL, [], 0}, 8) = 0
rt_sigaction(SIGSEGV, {0x8c7c4, ~[], SA_STACK|SA_RESTART|SA_SIGINFO}, NULL, 8) = 0
rt_sigaction(SIGUSR2, NULL, {SIG_DFL, [], 0}, 8) = 0
rt_sigaction(SIGUSR2, {0x8c7c4, ~[], SA_STACK|SA_RESTART|SA_SIGINFO}, NULL, 8) = 0
rt_sigaction(SIGPIPE, NULL, {SIG_DFL, [], 0}, 8) = 0
rt_sigaction(SIGPIPE, {0x8c7c4, ~[], SA_STACK|SA_RESTART|SA_SIGINFO}, NULL, 8) = 0
rt_sigaction(SIGALRM, NULL, {SIG_DFL, [], 0}, 8) = 0
rt_sigaction(SIGALRM, {0x8c7c4, ~[], SA_STACK|SA_RESTART|SA_SIGINFO}, NULL, 8) = 0
rt_sigaction(SIGTERM, NULL, {SIG_DFL, [], 0}, 8) = 0
rt_sigaction(SIGTERM, {0x8c7c4, ~[], SA_STACK|SA_RESTART|SA_SIGINFO}, NULL, 8) = 0
rt_sigaction(SIGSTKFLT, NULL, {SIG_DFL, [], 0}, 8) = 0
rt_sigaction(SIGSTKFLT, {0x8c7c4, ~[], SA_STACK|SA_RESTART|SA_SIGINFO}, NULL, 8) = 0
rt_sigaction(SIGCHLD, NULL, {SIG_DFL, [], 0}, 8) = 0
rt_sigaction(SIGCHLD, {0x8c7c4, ~[], SA_STACK|SA_RESTART|SA_SIGINFO}, NULL, 8) = 0
rt_sigaction(SIGURG, NULL, {SIG_DFL, [], 0}, 8) = 0
rt_sigaction(SIGURG, {0x8c7c4, ~[], SA_STACK|SA_RESTART|SA_SIGINFO}, NULL, 8) = 0
rt_sigaction(SIGXCPU, NULL, {SIG_DFL, [], 0}, 8) = 0
rt_sigaction(SIGXCPU, {0x8c7c4, ~[], SA_STACK|SA_RESTART|SA_SIGINFO}, NULL, 8) = 0
rt_sigaction(SIGXFSZ, NULL, {SIG_DFL, [], 0}, 8) = 0
rt_sigaction(SIGXFSZ, {0x8c7c4, ~[], SA_STACK|SA_RESTART|SA_SIGINFO}, NULL, 8) = 0
rt_sigaction(SIGVTALRM, NULL, {SIG_DFL, [], 0}, 8) = 0
rt_sigaction(SIGVTALRM, {0x8c7c4, ~[], SA_STACK|SA_RESTART|SA_SIGINFO}, NULL, 8) = 0
rt_sigaction(SIGPROF, NULL, {SIG_DFL, [], 0}, 8) = 0
rt_sigaction(SIGPROF, {0x8c7c4, ~[], SA_STACK|SA_RESTART|SA_SIGINFO}, NULL, 8) = 0
rt_sigaction(SIGWINCH, NULL, {SIG_DFL, [], 0}, 8) = 0
rt_sigaction(SIGWINCH, {0x8c7c4, ~[], SA_STACK|SA_RESTART|SA_SIGINFO}, NULL, 8) = 0
rt_sigaction(SIGIO, NULL, {SIG_DFL, [], 0}, 8) = 0
rt_sigaction(SIGIO, {0x8c7c4, ~[], SA_STACK|SA_RESTART|SA_SIGINFO}, NULL, 8) = 0
rt_sigaction(SIGPWR, NULL, {SIG_DFL, [], 0}, 8) = 0
rt_sigaction(SIGPWR, {0x8c7c4, ~[], SA_STACK|SA_RESTART|SA_SIGINFO}, NULL, 8) = 0
rt_sigaction(SIGSYS, NULL, {SIG_DFL, [], 0}, 8) = 0
rt_sigaction(SIGSYS, {0x8c7c4, ~[], SA_STACK|SA_RESTART|SA_SIGINFO}, NULL, 8) = 0
rt_sigaction(SIGRTMIN, NULL, {0x4ffe43ac, [], SA_RESTORER|SA_SIGINFO, 0x4febe800}, 8) = 0
rt_sigaction(SIGRTMIN, NULL, {0x4ffe43ac, [], SA_RESTORER|SA_SIGINFO, 0x4febe800}, 8) = 0
rt_sigaction(SIGRTMIN, {0x4ffe43ac, [], SA_RESTORER|SA_STACK|SA_SIGINFO, 0x4febe800}, NULL, 8) = 0
rt_sigaction(SIGRT_1, NULL, {0x4ffe4494, [], SA_RESTORER|SA_RESTART|SA_SIGINFO, 0x4febe800}, 8) = 0
rt_sigaction(SIGRT_1, NULL, {0x4ffe4494, [], SA_RESTORER|SA_RESTART|SA_SIGINFO, 0x4febe800}, 8) = 0
rt_sigaction(SIGRT_1, {0x4ffe4494, [], SA_RESTORER|SA_STACK|SA_RESTART|SA_SIGINFO, 0x4febe800}, NULL, 8) = 0
rt_sigaction(SIGRT_2, NULL, {SIG_DFL, [], 0}, 8) = 0
rt_sigaction(SIGRT_2, {0x8c7c4, ~[], SA_STACK|SA_RESTART|SA_SIGINFO}, NULL, 8) = 0
rt_sigaction(SIGRT_3, NULL, {SIG_DFL, [], 0}, 8) = 0
rt_sigaction(SIGRT_3, {0x8c7c4, ~[], SA_STACK|SA_RESTART|SA_SIGINFO}, NULL, 8) = 0
rt_sigaction(SIGRT_4, NULL, {SIG_DFL, [], 0}, 8) = 0
rt_sigaction(SIGRT_4, {0x8c7c4, ~[], SA_STACK|SA_RESTART|SA_SIGINFO}, NULL, 8) = 0
rt_sigaction(SIGRT_5, NULL, {SIG_DFL, [], 0}, 8) = 0
rt_sigaction(SIGRT_5, {0x8c7c4, ~[], SA_STACK|SA_RESTART|SA_SIGINFO}, NULL, 8) = 0
rt_sigaction(SIGRT_6, NULL, {SIG_DFL, [], 0}, 8) = 0
rt_sigaction(SIGRT_6, {0x8c7c4, ~[], SA_STACK|SA_RESTART|SA_SIGINFO}, NULL, 8) = 0
rt_sigaction(SIGRT_7, NULL, {SIG_DFL, [], 0}, 8) = 0
rt_sigaction(SIGRT_7, {0x8c7c4, ~[], SA_STACK|SA_RESTART|SA_SIGINFO}, NULL, 8) = 0
rt_sigaction(SIGRT_8, NULL, {SIG_DFL, [], 0}, 8) = 0
rt_sigaction(SIGRT_8, {0x8c7c4, ~[], SA_STACK|SA_RESTART|SA_SIGINFO}, NULL, 8) = 0
rt_sigaction(SIGRT_9, NULL, {SIG_DFL, [], 0}, 8) = 0
rt_sigaction(SIGRT_9, {0x8c7c4, ~[], SA_STACK|SA_RESTART|SA_SIGINFO}, NULL, 8) = 0
rt_sigaction(SIGRT_10, NULL, {SIG_DFL, [], 0}, 8) = 0
rt_sigaction(SIGRT_10, {0x8c7c4, ~[], SA_STACK|SA_RESTART|SA_SIGINFO}, NULL, 8) = 0
rt_sigaction(SIGRT_11, NULL, {SIG_DFL, [], 0}, 8) = 0
rt_sigaction(SIGRT_11, {0x8c7c4, ~[], SA_STACK|SA_RESTART|SA_SIGINFO}, NULL, 8) = 0
rt_sigaction(SIGRT_12, NULL, {SIG_DFL, [], 0}, 8) = 0
rt_sigaction(SIGRT_12, {0x8c7c4, ~[], SA_STACK|SA_RESTART|SA_SIGINFO}, NULL, 8) = 0
rt_sigaction(SIGRT_13, NULL, {SIG_DFL, [], 0}, 8) = 0
rt_sigaction(SIGRT_13, {0x8c7c4, ~[], SA_STACK|SA_RESTART|SA_SIGINFO}, NULL, 8) = 0
rt_sigaction(SIGRT_14, NULL, {SIG_DFL, [], 0}, 8) = 0
rt_sigaction(SIGRT_14, {0x8c7c4, ~[], SA_STACK|SA_RESTART|SA_SIGINFO}, NULL, 8) = 0
rt_sigaction(SIGRT_15, NULL, {SIG_DFL, [], 0}, 8) = 0
rt_sigaction(SIGRT_15, {0x8c7c4, ~[], SA_STACK|SA_RESTART|SA_SIGINFO}, NULL, 8) = 0
rt_sigaction(SIGRT_16, NULL, {SIG_DFL, [], 0}, 8) = 0
rt_sigaction(SIGRT_16, {0x8c7c4, ~[], SA_STACK|SA_RESTART|SA_SIGINFO}, NULL, 8) = 0
rt_sigaction(SIGRT_17, NULL, {SIG_DFL, [], 0}, 8) = 0
rt_sigaction(SIGRT_17, {0x8c7c4, ~[], SA_STACK|SA_RESTART|SA_SIGINFO}, NULL, 8) = 0
rt_sigaction(SIGRT_18, NULL, {SIG_DFL, [], 0}, 8) = 0
rt_sigaction(SIGRT_18, {0x8c7c4, ~[], SA_STACK|SA_RESTART|SA_SIGINFO}, NULL, 8) = 0
rt_sigaction(SIGRT_19, NULL, {SIG_DFL, [], 0}, 8) = 0
rt_sigaction(SIGRT_19, {0x8c7c4, ~[], SA_STACK|SA_RESTART|SA_SIGINFO}, NULL, 8) = 0
rt_sigaction(SIGRT_20, NULL, {SIG_DFL, [], 0}, 8) = 0
rt_sigaction(SIGRT_20, {0x8c7c4, ~[], SA_STACK|SA_RESTART|SA_SIGINFO}, NULL, 8) = 0
rt_sigaction(SIGRT_21, NULL, {SIG_DFL, [], 0}, 8) = 0
rt_sigaction(SIGRT_21, {0x8c7c4, ~[], SA_STACK|SA_RESTART|SA_SIGINFO}, NULL, 8) = 0
rt_sigaction(SIGRT_22, NULL, {SIG_DFL, [], 0}, 8) = 0
rt_sigaction(SIGRT_22, {0x8c7c4, ~[], SA_STACK|SA_RESTART|SA_SIGINFO}, NULL, 8) = 0
rt_sigaction(SIGRT_23, NULL, {SIG_DFL, [], 0}, 8) = 0
rt_sigaction(SIGRT_23, {0x8c7c4, ~[], SA_STACK|SA_RESTART|SA_SIGINFO}, NULL, 8) = 0
rt_sigaction(SIGRT_24, NULL, {SIG_DFL, [], 0}, 8) = 0
rt_sigaction(SIGRT_24, {0x8c7c4, ~[], SA_STACK|SA_RESTART|SA_SIGINFO}, NULL, 8) = 0
rt_sigaction(SIGRT_25, NULL, {SIG_DFL, [], 0}, 8) = 0
rt_sigaction(SIGRT_25, {0x8c7c4, ~[], SA_STACK|SA_RESTART|SA_SIGINFO}, NULL, 8) = 0
rt_sigaction(SIGRT_26, NULL, {SIG_DFL, [], 0}, 8) = 0
rt_sigaction(SIGRT_26, {0x8c7c4, ~[], SA_STACK|SA_RESTART|SA_SIGINFO}, NULL, 8) = 0
rt_sigaction(SIGRT_27, NULL, {SIG_DFL, [], 0}, 8) = 0
rt_sigaction(SIGRT_27, {0x8c7c4, ~[], SA_STACK|SA_RESTART|SA_SIGINFO}, NULL, 8) = 0
rt_sigaction(SIGRT_28, NULL, {SIG_DFL, [], 0}, 8) = 0
rt_sigaction(SIGRT_28, {0x8c7c4, ~[], SA_STACK|SA_RESTART|SA_SIGINFO}, NULL, 8) = 0
rt_sigaction(SIGRT_29, NULL, {SIG_DFL, [], 0}, 8) = 0
rt_sigaction(SIGRT_29, {0x8c7c4, ~[], SA_STACK|SA_RESTART|SA_SIGINFO}, NULL, 8) = 0
rt_sigaction(SIGRT_30, NULL, {SIG_DFL, [], 0}, 8) = 0
rt_sigaction(SIGRT_30, {0x8c7c4, ~[], SA_STACK|SA_RESTART|SA_SIGINFO}, NULL, 8) = 0
rt_sigaction(SIGRT_31, NULL, {SIG_DFL, [], 0}, 8) = 0
rt_sigaction(SIGRT_31, {0x8c7c4, ~[], SA_STACK|SA_RESTART|SA_SIGINFO}, NULL, 8) = 0
rt_sigaction(SIGRT_32, NULL, {SIG_DFL, [], 0}, 8) = 0
rt_sigaction(SIGRT_32, {0x8c7c4, ~[], SA_STACK|SA_RESTART|SA_SIGINFO}, NULL, 8) = 0
clock_gettime(CLOCK_MONOTONIC, {56447, 206678162}) = 0
clock_gettime(CLOCK_MONOTONIC, {56447, 207256565}) = 0
brk(0)                                  = 0x159e000
brk(0x15bf000)                          = 0x15bf000
rt_sigprocmask(SIG_SETMASK, ~[RTMIN RT_1], [], 8) = 0
mmap2(NULL, 8388608, PROT_READ|PROT_WRITE, MAP_PRIVATE|MAP_ANONYMOUS|MAP_STACK, -1, 0) = 0xb666f000
mprotect(0xb666f000, 4096, PROT_NONE)   = 0
clone(child_stack=0xb6e6df88, flags=CLONE_VM|CLONE_FS|CLONE_FILES|CLONE_SIGHAND|CLONE_THREAD|CLONE_SYSVSEM|CLONE_SETTLS|CLONE_PARENT_SETTID|CLONE_CHILD_CLEARTID, parent_tidptr=0xb6e6e4b8, tls=0xb6e6e910, child_tidptr=0xb6e6e4b8) = 19370
rt_sigprocmask(SIG_SETMASK, [], NULL, 8) = 0
mmap2(NULL, 262144, PROT_READ|PROT_WRITE, MAP_PRIVATE|MAP_ANONYMOUS, -1, 0) = 0xb662f000
clock_gettime(CLOCK_MONOTONIC, {56447, 214146643}) = 0
rt_sigprocmask(SIG_SETMASK, ~[RTMIN RT_1], [], 8) = 0
mmap2(NULL, 8388608, PROT_READ|PROT_WRITE, MAP_PRIVATE|MAP_ANONYMOUS|MAP_STACK, -1, 0) = 0xb5e2f000
mprotect(0xb5e2f000, 4096, PROT_NONE)   = 0
clone(child_stack=0xb662df88, flags=CLONE_VM|CLONE_FS|CLONE_FILES|CLONE_SIGHAND|CLONE_THREAD|CLONE_SYSVSEM|CLONE_SETTLS|CLONE_PARENT_SETTID|CLONE_CHILD_CLEARTID, parent_tidptr=0xb662e4b8, tls=0xb662e910, child_tidptr=0xb662e4b8) = 19371
rt_sigprocmask(SIG_SETMASK, [], NULL, 8) = 0
futex(0x1586b3c, FUTEX_WAIT, 0, NULL)   = 0
futex(0x117205a4, FUTEX_WAKE, 1)        = 1
openat(AT_FDCWD, "/proc/sys/net/core/somaxconn", O_RDONLY|O_LARGEFILE|O_CLOEXEC) = 3
read(3, "128\n", 4096)                  = 4
read(3, "", 4092)                       = 0
close(3)                                = 0
socket(PF_INET, SOCK_STREAM, IPPROTO_TCP) = 3
close(3)                                = 0
socket(PF_INET6, SOCK_STREAM, IPPROTO_TCP) = 3
setsockopt(3, SOL_IPV6, IPV6_V6ONLY, [1], 4) = 0
bind(3, {sa_family=AF_INET6, sin6_port=htons(0), inet_pton(AF_INET6, "::1", &sin6_addr), sin6_flowinfo=0, sin6_scope_id=0}, 28) = 0
socket(PF_INET6, SOCK_STREAM, IPPROTO_TCP) = 4
setsockopt(4, SOL_IPV6, IPV6_V6ONLY, [0], 4) = 0
bind(4, {sa_family=AF_INET6, sin6_port=htons(0), inet_pton(AF_INET6, "::ffff:127.0.0.1", &sin6_addr), sin6_flowinfo=0, sin6_scope_id=0}, 28) = 0
close(4)                                = 0
close(3)                                = 0
futex(0x117205a4, FUTEX_WAKE, 1)        = 1
clock_gettime(CLOCK_REALTIME, {1467715989, 620534261}) = 0
ioctl(1, TCGETS, 0x1173feb4)            = -1 ENOTTY (Inappropriate ioctl for device)
futex(0x117205a4, FUTEX_WAKE, 1)        = 1
stat64("/usr/local/bin/adduser", 0x1174c924) = -1 ENOENT (No such file or directory)
stat64("/usr/bin/adduser", 0x1174c9b4)  = -1 ENOENT (No such file or directory)
stat64("/bin/adduser", 0x1174ca44)      = -1 ENOENT (No such file or directory)
stat64("/usr/local/sbin/adduser", 0x1174cad4) = -1 ENOENT (No such file or directory)
stat64("/usr/sbin/adduser", {st_mode=S_IFREG|0755, st_size=574464, ...}) = 0
lstat64("/usr", {st_mode=S_IFDIR|0755, st_size=1024, ...}) = 0
lstat64("/usr/sbin", {st_mode=S_IFDIR|0755, st_size=3072, ...}) = 0
lstat64("/usr/sbin/adduser", {st_mode=S_IFLNK|0777, st_size=19, ...}) = 0
readlinkat(AT_FDCWD, "/usr/sbin/adduser", "/bin/busybox.nosuid", 128) = 19
lstat64("/bin", {st_mode=S_IFDIR|0755, st_size=3072, ...}) = 0
lstat64("/bin/busybox.nosuid", {st_mode=S_IFREG|0755, st_size=574464, ...}) = 0
stat64("/usr/local/bin/useradd", 0x1174cec4) = -1 ENOENT (No such file or directory)
stat64("/usr/bin/useradd", 0x1174cf54)  = -1 ENOENT (No such file or directory)
stat64("/bin/useradd", 0x1174cfe4)      = -1 ENOENT (No such file or directory)
stat64("/usr/local/sbin/useradd", 0x1174d074) = -1 ENOENT (No such file or directory)
stat64("/usr/sbin/useradd", 0x1174d104) = -1 ENOENT (No such file or directory)
stat64("/sbin/useradd", 0x1174d194)     = -1 ENOENT (No such file or directory)
stat64("/usr/local/bin/addgroup", 0x1174d224) = -1 ENOENT (No such file or directory)
stat64("/usr/bin/addgroup", 0x1174d2b4) = -1 ENOENT (No such file or directory)
stat64("/bin/addgroup", 0x1174d344)     = -1 ENOENT (No such file or directory)
stat64("/usr/local/sbin/addgroup", 0x1174d3d4) = -1 ENOENT (No such file or directory)
stat64("/usr/sbin/addgroup", {st_mode=S_IFREG|0755, st_size=574464, ...}) = 0
lstat64("/usr", {st_mode=S_IFDIR|0755, st_size=1024, ...}) = 0
lstat64("/usr/sbin", {st_mode=S_IFDIR|0755, st_size=3072, ...}) = 0
lstat64("/usr/sbin/addgroup", {st_mode=S_IFLNK|0777, st_size=19, ...}) = 0
readlinkat(AT_FDCWD, "/usr/sbin/addgroup", "/bin/busybox.nosuid", 128) = 19
lstat64("/bin", {st_mode=S_IFDIR|0755, st_size=3072, ...}) = 0
lstat64("/bin/busybox.nosuid", {st_mode=S_IFREG|0755, st_size=574464, ...}) = 0
stat64("/usr/local/bin/groupadd", 0x1174d7c4) = -1 ENOENT (No such file or directory)
stat64("/usr/bin/groupadd", 0x1174d854) = -1 ENOENT (No such file or directory)
stat64("/bin/groupadd", 0x1174d8e4)     = -1 ENOENT (No such file or directory)
stat64("/usr/local/sbin/groupadd", 0x1174d974) = -1 ENOENT (No such file or directory)
stat64("/usr/sbin/groupadd", 0x1174da04) = -1 ENOENT (No such file or directory)
stat64("/sbin/groupadd", 0x1174da94)    = -1 ENOENT (No such file or directory)
getrandom(0x1173fd44, 1, GRND_NONBLOCK) = -1 ENOSYS (Function not implemented)
openat(AT_FDCWD, "/dev/urandom", O_RDONLY|O_LARGEFILE|O_CLOEXEC) = 3
read(3, "\20Z\240r\300n~K\270\261\255mzlSgK\352\2310\342^w\230e=o\244]\341@\334"..., 4096) = 4096
futex(0x117205a4, FUTEX_WAKE, 1)        = 1
futex(0x117205a4, FUTEX_WAKE, 1)        = 1
futex(0x117205a4, FUTEX_WAKE, 1)        = 1
futex(0x117205a4, FUTEX_WAKE, 1)        = 1
mmap2(0x11800000, 1048576, PROT_READ|PROT_WRITE, MAP_PRIVATE|MAP_FIXED|MAP_ANONYMOUS, -1, 0) = 0x11800000
mmap2(0x116e0000, 65536, PROT_READ|PROT_WRITE, MAP_PRIVATE|MAP_FIXED|MAP_ANONYMOUS, -1, 0) = 0x116e0000
clock_gettime(CLOCK_MONOTONIC, {56447, 379770950}) = 0
futex(0x117205a4, FUTEX_WAKE, 1)        = 1
futex(0x117205a4, FUTEX_WAKE, 1)        = 1
futex(0x117205a4, FUTEX_WAKE, 1)        = 1
futex(0x117205a4, FUTEX_WAKE, 1)        = 1
futex(0x117205a4, FUTEX_WAKE, 1)        = 1
futex(0x117205a4, FUTEX_WAKE, 1)        = 1
futex(0x117205a4, FUTEX_WAKE, 1)        = 1
futex(0x117205a4, FUTEX_WAKE, 1)        = 1
futex(0x117205a4, FUTEX_WAKE, 1)        = 1
futex(0x117205a4, FUTEX_WAKE, 1)        = 1
futex(0x117205a4, FUTEX_WAKE, 1)        = 1
futex(0x117205a4, FUTEX_WAKE, 1)        = 1
futex(0x117205a4, FUTEX_WAKE, 1)        = 1
futex(0x1586b3c, FUTEX_WAIT, 0, NULL)   = 0
clock_gettime(CLOCK_MONOTONIC, {56447, 417956042}) = 0
clock_gettime(CLOCK_MONOTONIC, {56447, 420679153}) = 0
futex(0x11720aa4, FUTEX_WAKE, 1)        = 1
futex(0x11720aa4, FUTEX_WAKE, 1)        = 1
futex(0x11720aa4, FUTEX_WAKE, 1)        = 1
mmap2(0x11900000, 1048576, PROT_READ|PROT_WRITE, MAP_PRIVATE|MAP_FIXED|MAP_ANONYMOUS, -1, 0) = 0x11900000
mmap2(0x116d0000, 65536, PROT_READ|PROT_WRITE, MAP_PRIVATE|MAP_FIXED|MAP_ANONYMOUS, -1, 0) = 0x116d0000
clock_gettime(CLOCK_MONOTONIC, {56447, 490202947}) = 0
futex(0x11720aa4, FUTEX_WAKE, 1)        = 1
capget({0 /* _LINUX_CAPABILITY_VERSION_??? */, 0}, NULL) = 0
openat(AT_FDCWD, "/proc/sys/kernel/cap_last_cap", O_RDONLY|O_LARGEFILE|O_CLOEXEC) = 4
read(4, "36\n", 11)                     = 3
close(4)                                = 0
openat(AT_FDCWD, "/sys/kernel/mm/hugepages", O_RDONLY|O_LARGEFILE|O_CLOEXEC) = -1 ENOENT (No such file or directory)
futex(0x11720aa4, FUTEX_WAKE, 1)        = 1
futex(0x11720aa4, FUTEX_WAKE, 1)        = 1
futex(0x11720aa4, FUTEX_WAKE, 1)        = 1
futex(0x11720aa4, FUTEX_WAKE, 1)        = 1
futex(0x11720aa4, FUTEX_WAKE, 1)        = 1
futex(0x11720aa4, FUTEX_WAKE, 1)        = 1
futex(0x11720aa4, FUTEX_WAKE, 1)        = 1
futex(0x11720aa4, FUTEX_WAKE, 1)        = 1
futex(0x11720aa4, FUTEX_WAKE, 1)        = 1
futex(0x11720aa4, FUTEX_WAKE, 1)        = 1
futex(0x11720aa4, FUTEX_WAKE, 1)        = 1
futex(0x11720aa4, FUTEX_WAKE, 1)        = 1
futex(0x11720aa4, FUTEX_WAKE, 1)        = 1
futex(0x11720aa4, FUTEX_WAKE, 1)        = 1
futex(0x11720aa4, FUTEX_WAKE, 1)        = 1
futex(0x11720aa4, FUTEX_WAKE, 1)        = 1
futex(0x11720aa4, FUTEX_WAKE, 1)        = 1
futex(0x11720aa4, FUTEX_WAKE, 1)        = 1
futex(0x11720aa4, FUTEX_WAKE, 1)        = 1
futex(0x11720aa4, FUTEX_WAKE, 1)        = 1
futex(0x11720aa4, FUTEX_WAKE, 1)        = 1
futex(0x11720aa4, FUTEX_WAKE, 1)        = 1
futex(0x11720aa4, FUTEX_WAKE, 1)        = 1
futex(0x11720aa4, FUTEX_WAKE, 1)        = 1
futex(0x11720aa4, FUTEX_WAKE, 1)        = 1
futex(0x11720aa4, FUTEX_WAKE, 1)        = 1
futex(0x11720aa4, FUTEX_WAKE, 1)        = 1
futex(0x11720aa4, FUTEX_WAKE, 1)        = 1
futex(0x11720aa4, FUTEX_WAKE, 1)        = 1
futex(0x11720aa4, FUTEX_WAKE, 1)        = 1
futex(0x11720aa4, FUTEX_WAKE, 1)        = 1
futex(0x11720aa4, FUTEX_WAKE, 1)        = 1
futex(0x11720aa4, FUTEX_WAKE, 1)        = 1
futex(0x11720aa4, FUTEX_WAKE, 1)        = 1
futex(0x11720aa4, FUTEX_WAKE, 1)        = 1
futex(0x11720aa4, FUTEX_WAKE, 1)        = 1
futex(0x11720aa4, FUTEX_WAKE, 1)        = 1
futex(0x11720aa4, FUTEX_WAKE, 1)        = 1
futex(0x11720aa4, FUTEX_WAKE, 1)        = 1
futex(0x11720aa4, FUTEX_WAKE, 1)        = 1
futex(0x11720aa4, FUTEX_WAKE, 1)        = 1
futex(0x11720aa4, FUTEX_WAKE, 1)        = 1
futex(0x11720aa4, FUTEX_WAKE, 1)        = 1
futex(0x11720aa4, FUTEX_WAKE, 1)        = 1
futex(0x11720aa4, FUTEX_WAKE, 1)        = 1
futex(0x11720aa4, FUTEX_WAKE, 1)        = 1
futex(0x11720aa4, FUTEX_WAKE, 1)        = 1
futex(0x11720aa4, FUTEX_WAKE, 1)        = 1
futex(0x11720aa4, FUTEX_WAKE, 1)        = 1
futex(0x11720aa4, FUTEX_WAKE, 1)        = 1
futex(0x11720aa4, FUTEX_WAKE, 1)        = 1
futex(0x11720aa4, FUTEX_WAKE, 1)        = 1
futex(0x11720aa4, FUTEX_WAKE, 1)        = 1
futex(0x11720aa4, FUTEX_WAKE, 1)        = 1
futex(0x11720aa4, FUTEX_WAKE, 1)        = 1
futex(0x11720aa4, FUTEX_WAKE, 1)        = 1
futex(0x11720aa4, FUTEX_WAKE, 1)        = 1
futex(0x11720aa4, FUTEX_WAKE, 1)        = 1
clock_gettime(CLOCK_REALTIME, {1467715989, 957902260}) = 0
futex(0x11720aa4, FUTEX_WAKE, 1)        = 1
futex(0x11720aa4, FUTEX_WAKE, 1)        = 1
futex(0x11720aa4, FUTEX_WAKE, 1)        = 1
mmap2(0x11a00000, 1048576, PROT_READ|PROT_WRITE, MAP_PRIVATE|MAP_FIXED|MAP_ANONYMOUS, -1, 0) = 0x11a00000
mmap2(0x116c0000, 65536, PROT_READ|PROT_WRITE, MAP_PRIVATE|MAP_FIXED|MAP_ANONYMOUS, -1, 0) = 0x116c0000
clock_gettime(CLOCK_MONOTONIC, {56447, 642831635}) = 0
futex(0x11720aa4, FUTEX_WAKE, 1)        = 1
futex(0x11720aa4, FUTEX_WAKE, 1)        = 1
futex(0x11720aa4, FUTEX_WAKE, 1)        = 1
futex(0x11720aa4, FUTEX_WAKE, 1)        = 1
mmap2(0x11b00000, 1048576, PROT_READ|PROT_WRITE, MAP_PRIVATE|MAP_FIXED|MAP_ANONYMOUS, -1, 0) = 0x11b00000
mmap2(0x116b0000, 65536, PROT_READ|PROT_WRITE, MAP_PRIVATE|MAP_FIXED|MAP_ANONYMOUS, -1, 0) = 0x116b0000
clock_gettime(CLOCK_MONOTONIC, {56447, 727766190}) = 0
futex(0x11720aa4, FUTEX_WAKE, 1)        = 1
futex(0x11720aa4, FUTEX_WAKE, 1)        = 1
futex(0x11720aa4, FUTEX_WAKE, 1)        = 1
clock_gettime(CLOCK_MONOTONIC, {56447, 763168177}) = 0
clock_gettime(CLOCK_MONOTONIC, {56447, 763921332}) = 0
clock_gettime(CLOCK_MONOTONIC, {56447, 764510812}) = 0
clock_gettime(CLOCK_MONOTONIC, {56447, 765194743}) = 0
clock_gettime(CLOCK_MONOTONIC, {56447, 765806682}) = 0
futex(0x11720aa4, FUTEX_WAKE, 1)        = 1
futex(0x1586b3c, FUTEX_WAIT, 0, NULL)   = 0
clock_gettime(CLOCK_MONOTONIC, {56447, 860956520}) = 0
clock_gettime(CLOCK_MONOTONIC, {56447, 880705923}) = 0
futex(0x11720aa4, FUTEX_WAKE, 1)        = 1
futex(0x11720aa4, FUTEX_WAKE, 1)        = 1
futex(0x11720aa4, FUTEX_WAKE, 1)        = 1
futex(0x1586b3c, FUTEX_WAIT, 0, NULL)   = 0
futex(0x11720aa4, FUTEX_WAKE, 1)        = 1
futex(0x11720aa4, FUTEX_WAKE, 1)        = 1
mmap2(NULL, 262144, PROT_READ|PROT_WRITE, MAP_PRIVATE|MAP_ANONYMOUS, -1, 0) = 0xb54c0000
clock_gettime(CLOCK_MONOTONIC, {56447, 995435643}) = 0
clock_gettime(CLOCK_MONOTONIC, {56448, 2293725}) = 0
futex(0x11720aa4, FUTEX_WAKE, 1)        = 1
futex(0x1586b3c, FUTEX_WAIT, 0, NULL)   = 0
futex(0x11720aa4, FUTEX_WAKE, 1)        = 1
clock_gettime(CLOCK_MONOTONIC, {56448, 54435878}) = 0
clock_gettime(CLOCK_MONOTONIC, {56448, 63147312}) = 0
futex(0x11720aa4, FUTEX_WAKE, 1)        = 1
clock_gettime(CLOCK_REALTIME, {1467715990, 441399598}) = 0
clock_gettime(CLOCK_REALTIME, {1467715990, 442578864}) = 0
futex(0x11720aa4, FUTEX_WAKE, 1)        = 1
futex(0x1586b3c, FUTEX_WAIT, 0, NULL)   = 0
uname({sysname="Linux", nodename="ts-pxa", ...}) = 0
uname({sysname="Linux", nodename="ts-pxa", ...}) = 0
futex(0x11720aa4, FUTEX_WAKE, 1)        = 1
clock_gettime(CLOCK_REALTIME, {1467715990, 486622753}) = 0
clock_gettime(CLOCK_REALTIME, {1467715990, 492660301}) = 0
clock_gettime(CLOCK_REALTIME, {1467715990, 493415610}) = 0
socket(PF_LOCAL, SOCK_DGRAM|SOCK_CLOEXEC|SOCK_NONBLOCK, 0) = 4
setsockopt(4, SOL_SOCKET, SO_BROADCAST, [1], 4) = 0
connect(4, {sa_family=AF_LOCAL, sun_path="/run/systemd/journal/socket"}, 30) = 0
epoll_create1(EPOLL_CLOEXEC)            = 5
epoll_ctl(5, EPOLL_CTL_ADD, 4, {EPOLLIN|EPOLLOUT|EPOLLRDHUP|EPOLLET, {u32=3041678312, u64=3041678312}}) = 0
getsockname(4, {sa_family=AF_LOCAL, NULL}, [2]) = 0
getpeername(4, {sa_family=AF_LOCAL, sun_path="/run/systemd/journal/socket"}, [30]) = 0
futex(0x11720aa4, FUTEX_WAKE, 1)        = 1
futex(0x1586b3c, FUTEX_WAIT, 0, NULL)   = 0
openat(AT_FDCWD, "/etc/docker/daemon.json", O_RDONLY|O_LARGEFILE|O_CLOEXEC) = -1 ENOENT (No such file or directory)
umask(022)                              = 022
umask(022)                              = 022
openat(AT_FDCWD, "/var/run/docker.pid", O_RDONLY|O_LARGEFILE|O_CLOEXEC) = -1 ENOENT (No such file or directory)
getpid()                                = 19369
openat(AT_FDCWD, "/var/run/docker.pid", O_WRONLY|O_CREAT|O_TRUNC|O_LARGEFILE|O_CLOEXEC, 0644) = 6
write(6, "19369", 5)                    = 5
close(6)                                = 0
unlinkat(AT_FDCWD, "/var/run/docker.sock", 0) = 0
umask(0777)                             = 022
socket(PF_LOCAL, SOCK_STREAM|SOCK_CLOEXEC|SOCK_NONBLOCK, 0) = 6
setsockopt(6, SOL_SOCKET, SO_BROADCAST, [1], 4) = 0
setsockopt(6, SOL_SOCKET, SO_REUSEADDR, [1], 4) = 0
bind(6, {sa_family=AF_LOCAL, sun_path="/var/run/docker.sock"}, 23) = 0
listen(6, 128)                          = 0
epoll_ctl(5, EPOLL_CTL_ADD, 6, {EPOLLIN|EPOLLOUT|EPOLLRDHUP|EPOLLET, {u32=3041678192, u64=3041678192}}) = 0
getsockname(6, {sa_family=AF_LOCAL, sun_path="/var/run/docker.sock"}, [23]) = 0
openat(AT_FDCWD, "/etc/group", O_RDONLY|O_LARGEFILE|O_CLOEXEC) = 7
read(7, "root:x:0:\ndaemon:x:1:\nbin:x:2:\ns"..., 4096) = 661
read(7, "", 3435)                       = 0
close(7)                                = 0
fchownat(AT_FDCWD, "/var/run/docker.sock", 0, 993, 0) = 0
fchmodat(AT_FDCWD, "/var/run/docker.sock", 0660) = 0
umask(022)                              = 0777
stat64("/etc/docker/key.json", {st_mode=S_IFREG|0600, st_size=281, ...}) = 0
geteuid32()                             = 0
uname({sysname="Linux", nodename="ts-pxa", ...}) = 0
rt_sigprocmask(SIG_SETMASK, ~[ILL TRAP BUS FPE SEGV STKFLT CHLD PROF RTMIN RT_1], NULL, 8) = 0
rt_sigprocmask(SIG_SETMASK, ~[ILL TRAP BUS FPE USR1 SEGV STKFLT CHLD PROF RTMIN RT_1], NULL, 8) = 0
futex(0x11720aa4, FUTEX_WAKE, 1)        = 1
futex(0x1586b3c, FUTEX_WAIT, 0, NULL)   = 0
clock_gettime(CLOCK_MONOTONIC, {56448, 195440808}) = 0
rt_sigprocmask(SIG_SETMASK, ~[RTMIN RT_1], ~[ILL TRAP BUS FPE KILL USR1 SEGV STKFLT CHLD STOP PROF RTMIN RT_1], 8) = 0
mmap2(NULL, 8388608, PROT_READ|PROT_WRITE, MAP_PRIVATE|MAP_ANONYMOUS|MAP_STACK, -1, 0) = 0xb4b00000
mprotect(0xb4b00000, 4096, PROT_NONE)   = 0
clone(child_stack=0xb52fef88, flags=CLONE_VM|CLONE_FS|CLONE_FILES|CLONE_SIGHAND|CLONE_THREAD|CLONE_SYSVSEM|CLONE_SETTLS|CLONE_PARENT_SETTID|CLONE_CHILD_CLEARTID, parent_tidptr=0xb52ff4b8, tls=0xb52ff910, child_tidptr=0xb52ff4b8) = 19373
rt_sigprocmask(SIG_SETMASK, ~[ILL TRAP BUS FPE KILL USR1 SEGV STKFLT CHLD STOP PROF RTMIN RT_1], NULL, 8) = 0
futex(0x1586b3c, FUTEX_WAIT, 0, NULLtime="2016-07-05T10:53:10.583228138Z" level=info msg="[graphdriver] using prior storage driver \"btrfs\"" 
time="2016-07-05T10:53:10.667334469Z" level=info msg="Graph migration to content-addressability took 0.00 seconds" 
time="2016-07-05T10:53:10.713232941Z" level=info msg="Firewalld running: false" 
panic: runtime error: index out of range

goroutine 1 [running]:
panic(0xd8ff38, 0x1171e008)
	/home/theodor/resin-devices/resin-ts/build/tmp/sysroots/x86_64-linux/usr/lib/arm-poky-linux-gnueabi/go/src/runtime/panic.go:464 +0x330
github.com/boltdb/bolt.(*Bucket).pageNode(0x11711f20, 0x6b720004, 0x0, 0x0, 0x0)
	/home/theodor/resin-devices/resin-ts/build/tmp/work/armv5e-poky-linux-gnueabi/docker/1.10.3+git20f81dde9bd97c86b2d0e33bbbf1388018611929-r0/git/vendor/src/github.com/boltdb/bolt/bucket.go:696 +0x2f8
github.com/boltdb/bolt.(*Cursor).search(0x11bf2eb4, 0x11bf2e1c, 0x1b, 0x1b, 0x6b720004, 0x0)
	/home/theodor/resin-devices/resin-ts/build/tmp/work/armv5e-poky-linux-gnueabi/docker/1.10.3+git20f81dde9bd97c86b2d0e33bbbf1388018611929-r0/git/vendor/src/github.com/boltdb/bolt/cursor.go:238 +0x30
github.com/boltdb/bolt.(*Cursor).seek(0x11bf2eb4, 0x11bf2e1c, 0x1b, 0x1b, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0, ...)
	/home/theodor/resin-devices/resin-ts/build/tmp/work/armv5e-poky-linux-gnueabi/docker/1.10.3+git20f81dde9bd97c86b2d0e33bbbf1388018611929-r0/git/vendor/src/github.com/boltdb/bolt/cursor.go:152 +0xd0
github.com/boltdb/bolt.(*Cursor).Seek(0x11bf2eb4, 0x11bf2e1c, 0x1b, 0x1b, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0)
	/home/theodor/resin-devices/resin-ts/build/tmp/work/armv5e-poky-linux-gnueabi/docker/1.10.3+git20f81dde9bd97c86b2d0e33bbbf1388018611929-r0/git/vendor/src/github.com/boltdb/bolt/cursor.go:111 +0x5c
github.com/docker/libkv/store/boltdb.(*BoltDB).List.func1(0x11912100, 0x0, 0x0)
	/home/theodor/resin-devices/resin-ts/build/tmp/work/armv5e-poky-linux-gnueabi/docker/1.10.3+git20f81dde9bd97c86b2d0e33bbbf1388018611929-r0/git/vendor/src/github.com/docker/libkv/store/boltdb/boltdb.go:285 +0x178
github.com/boltdb/bolt.(*DB).View(0x1178ec00, 0x11bf2f24, 0x0, 0x0)
	/home/theodor/resin-devices/resin-ts/build/tmp/work/armv5e-poky-linux-gnueabi/docker/1.10.3+git20f81dde9bd97c86b2d0e33bbbf1388018611929-r0/git/vendor/src/github.com/boltdb/bolt/db.go:578 +0xb0
github.com/docker/libkv/store/boltdb.(*BoltDB).List(0x11932480, 0x11711c00, 0x1b, 0x0, 0x0, 0x0, 0x0, 0x0)
	/home/theodor/resin-devices/resin-ts/build/tmp/work/armv5e-poky-linux-gnueabi/docker/1.10.3+git20f81dde9bd97c86b2d0e33bbbf1388018611929-r0/git/vendor/src/github.com/docker/libkv/store/boltdb/boltdb.go:299 +0x19c
github.com/docker/libnetwork/datastore.(*cache).kmap(0x117950c0, 0xb54c6158, 0x1188be00, 0x74ff8, 0x0, 0x0)
	/home/theodor/resin-devices/resin-ts/build/tmp/work/armv5e-poky-linux-gnueabi/docker/1.10.3+git20f81dde9bd97c86b2d0e33bbbf1388018611929-r0/git/vendor/src/github.com/docker/libnetwork/datastore/cache.go:43 +0x20c
github.com/docker/libnetwork/datastore.(*cache).list(0x117950c0, 0xb54c6158, 0x1188be00, 0x0, 0x0, 0x0, 0x0, 0x0)
	/home/theodor/resin-devices/resin-ts/build/tmp/work/armv5e-poky-linux-gnueabi/docker/1.10.3+git20f81dde9bd97c86b2d0e33bbbf1388018611929-r0/git/vendor/src/github.com/docker/libnetwork/datastore/cache.go:139 +0x48
github.com/docker/libnetwork/datastore.(*datastore).List(0x11bfeb70, 0x11711b20, 0x1b, 0xb54c6158, 0x1188be00, 0x0, 0x0, 0x0, 0x0, 0x0)
	/home/theodor/resin-devices/resin-ts/build/tmp/work/armv5e-poky-linux-gnueabi/docker/1.10.3+git20f81dde9bd97c86b2d0e33bbbf1388018611929-r0/git/vendor/src/github.com/docker/libnetwork/datastore/datastore.go:472 +0xd8
github.com/docker/libnetwork/drivers/bridge.(*driver).populateNetworks(0x118a1920, 0x0, 0x0)
	/home/theodor/resin-devices/resin-ts/build/tmp/work/armv5e-poky-linux-gnueabi/docker/1.10.3+git20f81dde9bd97c86b2d0e33bbbf1388018611929-r0/git/vendor/src/github.com/docker/libnetwork/drivers/bridge/bridge_store.go:49 +0x130
github.com/docker/libnetwork/drivers/bridge.(*driver).initStore(0x118a1920, 0x11834e20, 0x0, 0x0)
	/home/theodor/resin-devices/resin-ts/build/tmp/work/armv5e-poky-linux-gnueabi/docker/1.10.3+git20f81dde9bd97c86b2d0e33bbbf1388018611929-r0/git/vendor/src/github.com/docker/libnetwork/drivers/bridge/bridge_store.go:42 +0x33c
github.com/docker/libnetwork/drivers/bridge.(*driver).configure(0x118a1920, 0x11834e20, 0x0, 0x0)
	/home/theodor/resin-devices/resin-ts/build/tmp/work/armv5e-poky-linux-gnueabi/docker/1.10.3+git20f81dde9bd97c86b2d0e33bbbf1388018611929-r0/git/vendor/src/github.com/docker/libnetwork/drivers/bridge/bridge.go:395 +0x310
github.com/docker/libnetwork/drivers/bridge.Init(0xb54c5f68, 0x1188a060, 0x11834e20, 0x0, 0x0)
	/home/theodor/resin-devices/resin-ts/build/tmp/work/armv5e-poky-linux-gnueabi/docker/1.10.3+git20f81dde9bd97c86b2d0e33bbbf1388018611929-r0/git/vendor/src/github.com/docker/libnetwork/drivers/bridge/bridge.go:140 +0x58c
github.com/docker/libnetwork.initDrivers(0x1188a060, 0x0, 0x0)
	/home/theodor/resin-devices/resin-ts/build/tmp/work/armv5e-poky-linux-gnueabi/docker/1.10.3+git20f81dde9bd97c86b2d0e33bbbf1388018611929-r0/git/vendor/src/github.com/docker/libnetwork/drivers.go:20 +0x220
github.com/docker/libnetwork.New(0x118349a0, 0x5, 0x8, 0x0, 0x0, 0x0, 0x0)
	/home/theodor/resin-devices/resin-ts/build/tmp/work/armv5e-poky-linux-gnueabi/docker/1.10.3+git20f81dde9bd97c86b2d0e33bbbf1388018611929-r0/git/vendor/src/github.com/docker/libnetwork/controller.go:189 +0x464
github.com/docker/docker/daemon.(*Daemon).initNetworkController(0x1176e180, 0x1193c180, 0x0, 0x0, 0x0, 0x0)
	/home/theodor/resin-devices/resin-ts/build/tmp/work/armv5e-poky-linux-gnueabi/docker/1.10.3+git20f81dde9bd97c86b2d0e33bbbf1388018611929-r0/git/.gopath/src/github.com/docker/docker/daemon/daemon_unix.go:532 +0x9c
github.com/docker/docker/daemon.NewDaemon(0x1193c180, 0x1171ec08, 0x0, 0x0, 0x0)
	/home/theodor/resin-devices/resin-ts/build/tmp/work/armv5e-poky-linux-gnueabi/docker/1.10.3+git20f81dde9bd97c86b2d0e33bbbf1388018611929-r0/git/.gopath/src/github.com/docker/docker/daemon/daemon.go:761 +0x1dfc
main.(*DaemonCli).CmdDaemon(0x11950820, 0x1170a108, 0x0, 0x0, 0x0, 0x0)
	/home/theodor/resin-devices/resin-ts/build/tmp/work/armv5e-poky-linux-gnueabi/docker/1.10.3+git20f81dde9bd97c86b2d0e33bbbf1388018611929-r0/git/docker/daemon.go:252 +0x1430
reflect.callMethod(0x11952ba0, 0x11bf3e04)
	/home/theodor/resin-devices/resin-ts/build/tmp/sysroots/x86_64-linux/usr/lib/arm-poky-linux-gnueabi/go/src/reflect/value.go:631 +0x1ac
reflect.methodValueCall(0x1170a108, 0x0, 0x0, 0x1, 0x11952ba0, 0x0, 0x0, 0x11952ba0, 0x1171eb40, 0x1efcc, ...)
	/home/theodor/resin-devices/resin-ts/build/tmp/sysroots/x86_64-linux/usr/lib/arm-poky-linux-gnueabi/go/src/reflect/asm_arm.s:29 +0x38
github.com/docker/docker/cli.(*Cli).Run(0x11952b20, 0x1170a108, 0x1, 0x1, 0x0, 0x0)
	/home/theodor/resin-devices/resin-ts/build/tmp/work/armv5e-poky-linux-gnueabi/docker/1.10.3+git20f81dde9bd97c86b2d0e33bbbf1388018611929-r0/git/.gopath/src/github.com/docker/docker/cli/cli.go:89 +0x2e8
main.main()
	/home/theodor/resin-devices/resin-ts/build/tmp/work/armv5e-poky-linux-gnueabi/docker/1.10.3+git20f81dde9bd97c86b2d0e33bbbf1388018611929-r0/git/docker/docker.go:63 +0x40c
 <unfinished ...>
+++ exited with 2 +++
`

--- a/hack/vendor.sh
+++ b/hack/vendor.sh
@@ -88,7 +88,7 @@ clone git github.com/deckarep/golang-set ef32fa3046d9f249d399f98ebaf9be944430fd1
 clone git github.com/coreos/etcd 3a49cbb769ebd8d1dd25abb1e83386e9883a5707
 clone git github.com/ugorji/go f1f1a805ed361a0e078bb537e4ea78cd37dcf065
 clone git github.com/hashicorp/consul v0.5.2
-clone git github.com/boltdb/bolt v1.2.1
+clone git github.com/boltdb/bolt fff57c100f4dea1905678da7e90d92429dff2904
 clone git github.com/miekg/dns 75e6e86cc601825c5dbcd4e0c209eab180997cd7
 
 # get graph and distribution packages

